internal/starlark/starlark.go

```diff
@@ -8,7 +8,7 @@ import (
 
 type StarlarkFunctionHandle struct {
 	Code              string
-	FunctionSignature db.FunctionSignature
+	FunctionSignature db.EnumValue
```

Comment:
I don't really like that this is now untyped (ie we don't know which enum) -- Should we do some kind of conversion to a go-land specific enum type?